lib/net.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const net = require('net');
+
+const sock = require('./socket');
```

Comment:
Having `socket` as the name of the parameter of `createTransport()` is more important.